front-end-poc/src/Audit/AuditOverview/AuditOverview.tsx

```diff
@@ -118,7 +118,8 @@ const AuditOverview = () => {
           // temporary solution: for demo purposes we display the first 3 executions here
           if (response.data.total > 0 && latestSearches.length === 0) {
             let searches = [];
-            for (let i = 0; i < 3; i++) {
+            let maxSearches = Math.min(3, response.data.headers.length);
```

Comment:
Fixed this too... I was wondering why my "Last opened.." was always the `SkeletonInlineStripe` :-)

Reply:
Oops! :facepalm: